kits/phishing-triage/.env.example

```
@@ -0,0 +1,7 @@
+# Phishing Triage flow ID (from Lamatic Studio → your deployed flow)
+PHISHING_TRIAGE="PHISHING_TRIAGE Flow ID"
+
+# Lamatic project credentials (Studio → Settings)
+LAMATIC_API_URL="LAMATIC_API_URL"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)